webrender/src/renderer.rs

```diff
@@ -513,6 +530,22 @@ impl Renderer {
                                                            &mut device,
                                                            options.precache_shaders);
 
+        let ps_yuv_image = PrimitiveShader {
```

Comment:
Should be possible to create as a normal PrimitiveShader once the changes mentioned above land.